packages/node_modules/@webex/plugin-meetings/src/meeting/request.js

```diff
@@ -43,6 +43,11 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       sipUri, deviceUrl, locusUrl, resourceId, correlationId, ensureConversation, moderator, pin, moveToResource, roapMessage
     } = options;
 
+    LoggerProxy.logger.log(
+      `${MeetingRequest.name}#${this.joinMeeting.name} --> Joining a meeting`,
```

Comment:
I thought this would be less brittle than what we were doing. I can change it back if we would rather

Reply:
I like this! Want to change all the others in the plugin too? 😉